round-coord.js

```diff
@@ -1,6 +1,14 @@
-module.exports = function roundCoord(c, precision) {
-    return [
-        Math.round(c[0] / precision) * precision,
-        Math.round(c[1] / precision) * precision,
-    ];
-};
+module.exports = {
+    coordToInt: function coordToInt(c, precision) {
```

Comment:
I change this up a bit to shift things into ints before using them as keys, it does lead to smaller strings a bit. my intention with was fixing some mismatched node lookups, which I thought was some comparison issues with floats, but the issue remains and is probably related to some nodes that are being dropped from the graph? Still more do look into there.